<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestoreWalletPage</class>
 <widget class="QWidget" name="RestoreWalletPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	color:#000;
}

QLineEdit {
	background:#fff;
	color:#000;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>60</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonBack">
         <property name="minimumSize">
          <size>
           <width>43</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>43</width>
           <height>38</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#f8f8f8;
color:#000;
border-radius:6.4px;
border:1px solid #dedede;</string>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset resource="../paicoin.qrc">
           <normaloff>:/icons/back</normaloff>:/icons/back</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>60</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelTitle">
         <property name="minimumSize">
          <size>
           <width>424</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-bottom:20;</string>
         </property>
         <property name="text">
          <string>Paper Key</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="labelInfo">
         <property name="styleSheet">
          <string notr="true">margin-bottom:20;</string>
         </property>
         <property name="text">
          <string>Enter the Paper Key of the wallet you want to restore</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="lineEdit01">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit02">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit03">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit04">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit05">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit06">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit07">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit08">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit09">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit10">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit11">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit12">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButtonRestoreWallet">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#1972ea;
color:#fff;</string>
         </property>
         <property name="text">
          <string>Restore Wallet</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>35</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../paicoin.qrc"/>
 </resources>
 <connections/>
</ui>
